app/daemon.js

```diff
@@ -102,23 +156,32 @@ export function ensuresIPFSInitialised () {
 }
 
 /**
- * Returns the multiAddr usable to connect to the local dameon via API
+ * 5001 -> 5101
```

Comment:
Add few lines about `ensureAddressesConfigured`, why this should be called? what does it do?